### PR TITLE
WIP. pj-53-54, add posts(last, hot)

### PR DESCRIPTION
**Pull request by NykoEugen**

### CODE REVIEW

**Review by acman — APPROVED** [review]
👍

File: core/views.py [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
+from posts.models import Post
+
 
 def home(request: HttpRequest) -> HttpResponse:
-    return render(request, "home.html")
+    last_posts = Post.objects.filter(is_active=True).order_by("-updated_at")[:5]
+    most_hot = (
+        Post.objects.annotate(comment_count=Count("comments", distinct=True))
+        .order_by("-comment_count")
+        .first()
```
Comment (acman) [inline core/views.py]: [:5]

**Review by acman — COMMENTED** [review]
Looks good. Fix tests and go

**Comment by NykoEugen** [comment]
Somthing wrong with migrations

**Pull request merged**